src/Microsoft.VisualStudio.Composition.Analyzers/VSMEF001PropertyMustHaveSetter.cs

```diff
@@ -40,25 +40,25 @@ public class VSMEF001PropertyMustHaveSetter : DiagnosticAnalyzer
         public override void Initialize(AnalysisContext context)
         {
             context.EnableConcurrentExecution();
-            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.ReportDiagnostics);
+            context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.Analyze | GeneratedCodeAnalysisFlags.ReportDiagnostics);
 
-            context.RegisterCompilationStartAction(compilationContext =>
+            context.RegisterCompilationStartAction(context =>
```

Comment:
I don't understand why you would rename this parameter when it now collides with another parameter in the `AnalyzePropertyDeclaration` lambda. Maybe to make using the broader-scoped parameter impossible?
I'd prefer the more precisely named parameters to hiding outer ones.

Reply:
@AArnott No analyzer will ever need the outer context. This pattern is also followed extensively in dotnet/roslyn to shadow the outer contexts.